distrib/deb/control/control

```diff
@@ -0,0 +1,10 @@
+Package: [[package.name]]
+Version: [[version]]-[[package.revision]]
+Section: admin
+Priority: optional
+Depends: kura-core (>= 6.0.0~), kura-core(<< 7.0.0~)
```

Comment:
This not the only dependency of `kura-position`. There's a dependency also on `kura-networking` given that we can retrieve the position informations from the modem, right?

Given the [Debian documentation](https://www.debian.org/doc/debian-policy/ch-relationships.html), I would say that the correct way to model this would be via a `Suggests` field.

> **Suggests**
> This is used to declare that one package may be more useful with one or more others. Using this field tells the packaging system and the user that the listed packages are related to this one and can perhaps enhance its usefulness, but that installing this one without them is perfectly reasonable.